debian/rules.in

```diff
@@ -9,14 +9,36 @@
 # Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1
 
+CUDA_VER = 11.6
```

Comment:
why need a specific version?

Reply:
Got rid of that. But there's still the build dependency in debian/control .

Reply:
could we require a lower version? such as 11.1?